packages/android_alarm_manager/example/lib/main.dart

```diff
@@ -107,7 +107,7 @@ class _AlarmHomePageState extends State<_AlarmHomePage> {
     final textStyle = Theme.of(context).textTheme.headline4;
     return Scaffold(
       appBar: AppBar(
-        title: Text(widget.title),
+        title: Text(widget.title!),
```

Comment:
This is unsafe; you can't assert the existence of field that was declared as nullable.